userguide/tutorials/plugin_development.adoc

```diff
@@ -398,10 +399,83 @@ Note that if you don't need any custom configuration, make sure to delete the de
+* If `--destination` is not specified, the plugin is installed by default in the `var/tmp/bundles` directory
+* For detailed instructions, refer to the https://docs.killbill.io/latest/plugin_development.html#_deployment[deployment] section above.
+
+The steps for installing a custom Ruby plugin are very similar to that of installing a Java plugin. Instead of the jar path, the path of the package (tar.gz) needs to be specified. 
+
+So, a custom Ruby plugin can be installed using the `kpm install_ruby_plugin` command as shown below:
+
+[source,bash]
+kpm install_ruby_plugin '<plugin-key>' --from-source-file="<package_path>.tar.gz"  
+--destination="<path_to_install_plugin>"
+
+* Replace `plugin-key`,`<package_path>` and `<path_to_install_plugin>` with appropriate values. 
+
+==== Kill Bill Java/Ruby Plugins
 
-The standard way to deploy plugins is to rely on https://github.com/killbill/killbill-cloud/blob/master/kpm[KPM].
-The https://github.com/killbill/killbill-cloud/blob/831e3c4ca536788ce64c83dfbd09b68668dddb6d/kpm/README.adoc[KPM README] explains how to install KPM and also provides some guidance on how to deploy publicly released Kill Bill plugins.
+A Kill Bill plugin is a plugin that is maintained by the Kill Bill team. Such plugins have a key in the Kill Bill https://github.com/killbill/killbill-cloud/blob/master/kpm/lib/kpm/plugins_directory.yml[Plugin Directory]. So, in order to install such a plugin, its key needs to be specified.
+
+A Kill Bill Java plugin can be installed using the `kpm install_java_plugin` command as follows:
+
+[source,bash]
+kpm install_java_plugin '<plugin-key>'
+
+* Replace `plugin-key` with an appropriate value from the Kill Bill https://github.com/killbill/killbill-cloud/blob/master/kpm/lib/kpm/plugins_directory.yml[Plugin Directory]. 
+* For example, in order to install the stripe plugin, you can replace `plugin-key` with `stripe`
+
+Similarly, a Kill Bill Ruby plugin can be installed using the `kpm install_ruby_plugin` command as follows:
+
+[source,bash]
+kpm install_ruby_plugin '<plugin-key>'
+
+* As in the case of a Java plugin, you need to replace `plugin-key` with an appropriate value. 
+
+==== Third party Java/Ruby plugins
+
+Third-party plugins are plugins developed and maintained by a third party and not by the Kill Bill team.  Such plugins can be installed via KPM from their binary repositories (Maven Central, GitHub Packages and Cloudsmith).
+
+A third party Java plugin can be installed using the `kpm install_java_plugin` command as follows: 
+
+[source,bash]
+kpm install_java_plugin '<plugin-key>' 
+--group_id="<group id>"
+--artifact_id="<artifact_id>"
+--version="<version>"  
+
+* Replace `plugin-key` with the plugin name.
+* Replace `<group id>`, `<artifact_id>`, `<version>` with appropriate values from the binary repository.
+
+Similarly, a third party Ruby plugin can be installed using the `kpm install_ruby_plugin` command as follows: 
```

Comment:
You can remove Ruby instructions, as we don't officially support Ruby plugins at the moment.

Reply:
Will do. In addition, there are other places in the document that talk about Ruby plugins like https://docs.killbill.io/latest/plugin_development.html#_ruby_plugins, https://docs.killbill.io/latest/plugin_development.html#_deploying_by_hand

Should I delete these sections as well?

Reply:
> Should I delete these sections as well?
Yes, that would remove any confusion. Thanks!

Reply:
Fixed